build.rs

```diff
@@ -2,11 +2,15 @@ use std::env;
 use std::path::PathBuf;
 use std::process::Command;
 
+const DTRACE_SRC_DIR: &str = "_dtrace";
+
 // Set-ExecutionPolicy RemoteSigned –Scope Process
 // 'C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin\MSBuild.exe' opendtrace.sln /t:dtrace_dll:Rebuild /p:Configuration=Release /p:Platform=x64
 fn get_dtrace_libpath() -> PathBuf {
     let dir = env::var("CARGO_MANIFEST_DIR").unwrap();
-    std::path::Path::new(&dir).join("target\\dtrace\\build\\x64\\Release\\lib\\")
+    std::path::Path::new(&dir)
+        .join(DTRACE_SRC_DIR)
+        .join("dtrace/build/x64/Release/lib")
```

Comment:
Wrong path, evaluates to `libdtrace-rs\_dtrace\dtrace/build/x64/Release/lib`

Should be `libdtrace-rs\target\_dtrace/build/x64/Release/lib`